setup.py

```diff
@@ -141,6 +139,7 @@ def _set_builtin(name, value):
 )
 
 if cythonize:
```

Comment:
In terms of complexity, I don't know that there is much benefit any more of shipping the cython files. I would suggest considering avoiding this in the future.

Reply:
Yes, I looked into this at some length for the same reason.

@bjlittle and I discussed your comment in #108:

>I'm under no illusions that the approach I've taken in this PR is ideal - but it should be an effective stop-gap until some clear pattern emerges as the recommended approach.

... given we are now using cibuildwheel for releases, which will be the majority of builds. But we noted that cf-units is used/developed/built by a diverse set of parties, on a diverse set of architectures, so we decided that the cost/benefit wasn't worth it - the cost being potentially breaking a workflow, versus the benefit of a mild complexity reduction.

Reply:
:+1: as already discussed